### DISCUSSION TITLE
文章不显示在首页，只能从标签里面找到，默认主题里面的配置没动，需要改什么嘛

### DISCUSSION THREAD

**Discussion by cruvie** [hugo-next/hugo-theme-next#98] [Q&A] [answered]
<img width="1180" alt="iShot_2023-05-22_12 25 24" src="https://github.com/hugo-next/hugo-theme-next/assets/55651847/0aadab97-0fce-4c09-a83a-ef786fca5432">

**Comment by elkan1788** [comment, marked as answer]
需要把你放文章的文件夹名称添加到 `config.yaml` 配置文件中，里面有个参数如下：

```
# 需要显示文章的部分，即content目录下的文件夹名称
# Sections for show in home & archive page 
# and it's forlder name which under content
mainSections: ["releases", "demo"]
```
